indico/modules/events/timetable/templates/pdf/_program.html

```diff
@@ -0,0 +1,340 @@
+                {{ subcontribs }}
+            </ul>
+        </div>
+    {% else %}
+        <div class="session-block-content-box inherited-border-color">
+            <ul class="contrib-timeline">
+                {{ subcontribs }}
+            </ul>
+        </div>
+    {% endif %}
+{% endmacro %}
+
+{% macro render_subcontrib(subcontrib, config) %}
+    {{ render_child_contrib(subcontrib, config) }}
+{% endmacro %}
+
+{% macro render_entry_header(entry) %}
+    {% set object = entry.object %}
+    {% set sess = object.session %}
+    {% set type = entry.type.name %}
+
+    <h3>
+        {% if type == 'SESSION_BLOCK' %}
+            {{ render_block_title(object) }}
+        {% else %}
+            {{ object.title }}
+        {% endif %}
+    </h3>
+    <div class="session-block-title-info">
+        <p>
+            <span class="bold">
+                {% if type == 'BREAK' %}
+                    {% trans %}Break{% endtrans %}
+                {% elif type == 'CONTRIBUTION' %}
+                    {% trans %}Contribution{% endtrans %}
+                {% elif sess.is_poster %}
+                    {% trans %}Poster Session{% endtrans %}
+                {% else %}
+                    {% trans %}Session{% endtrans %}
+                {% endif %}
+            </span>
+        </p>
+        <p>
+            <span class="bold">{% trans %}Location{% endtrans %}:</span>
+            {{ render_place(entry) }}
+        </p>
+        {% set people = object.person_links if type == 'SESSION_BLOCK' else object.speakers %}
+        {% if type != 'BREAK' and people %}
+            <span class="bold">
+                {% if type == 'SESSION_BLOCK' %}
+                    {{ ngettext('Convener:', 'Conveners:', people|length) }}
+                {% else %}
+                    {{ ngettext('Speaker:', 'Speakers:', people|length) }}
+                {% endif %}
+                &nbsp;
+            </span>
+            {{ render_names(people, show_title=config.show_title, show_affiliation=config.show_affiliation) }}
+        {% endif %}
+    </div>
+{% endmacro %}
+
+{% macro render_block_date(entry, day, print_date_close_to_sessions) %}
+    {% if print_date_close_to_sessions %}
+        <p class="session-block-date">
+            <span class="bold">{{ day|format_skeleton('EEEEdMMMM', timezone=entry.event.timezone) }}</span>
+            <span class="timebox">
+                {{ format_interval(entry.start_dt.astimezone(entry.event.tzinfo), entry.end_dt.astimezone(entry.event.tzinfo), format='HHmm') }}
+            </span>
+        </p>
+    {% endif %}
+{% endmacro %}
+
+{% macro render_time(entry) %}
+    <div class="time">
+        <div class="time-from">
+            {{ entry.start_dt|format_time(timezone=entry.event.timezone) }}
+        </div>
+        <div class="time-divider">
+            {# Line between generated in CSS #}
+        </div>
+        <div class="time-divider">
+            {# Line between generated in CSS #}
+        </div>
```

Comment:
I would add a comment why we have two of them, assuming this is intentional. Because right now I'd assume the empty div is intentional, but I have no clue about it being there twice.

Reply:
And it might better like this in the end (just one place where you explain the need for it)

```html
{# blah blah blah #}
<div class="time-divider"></div>
<div class="time-divider"></div>
```